app/main.py

```diff
@@ -1,6 +1,39 @@
+import os
+import requests
+
+
+API_KEY = os.getenv("WEATHER_API_KEY")
+BASE_URL = "http://api.weatherapi.com/v1"
+FILTERING = "Paris"
+
+
 def get_weather() -> None:
-    # write your code here
-    pass
+    if API_KEY is not None:
```

Comment:
It's good practice to check if `API_KEY` is `None`, but you should also handle the case where the environment variable is an empty string. Consider using `if not API_KEY:` to cover both cases.